networkx/algorithms/operators/all.py

```diff
@@ -232,5 +238,6 @@ def intersection_all(graphs):
 
     R.add_nodes_from(node_intersection)
     R.add_edges_from(edge_intersection)
+    R.graph.clear()  # remove graph attributes from intersection graph
```

Comment:
Instead of removing the R.graph attributes, we can remove line 234 (`R.graph.update`). That avoids adding them just to remove them again.

Reply:
Ah yes, this is better!